packages/hash/api/src/model/ontology/property-type.model.ts

```diff
@@ -47,24 +63,31 @@ export default class {
     return new PropertyTypeModel({
       schema: inner as PropertyType,
       ownedById: identifier.ownedById,
+      createdById,
+      updatedById,
+      removedById,
     });
   }
 
   /**
    * Create a property type.
    *
-   * @param params.ownedById the id of the owner of the property type
-   * @param params.schema a `PropertyType`
+   * @param params.ownedById - the id of the owner of the property type
```

Comment:
```suggestion
   * @param params.ownedById - the id of the account that owns the property type
```

Reply:
Addressed in https://github.com/hashintel/hash/pull/1197/commits/51c3a539be68ca7bf1c7ffa8f13550e6f20efee4